### PR TITLE
Move web-stories-block to new structure

### LINKED ISSUE #6634: Packages: Move `web-stories-block` to new structure
## Task Description

The embed block should be moved to a new `@web-stories-wp/stories-block` package or similar.

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

The embed block should be moved to a new @web-stories-wp/stories-block package or similar.

### QA

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6634

### CODE REVIEW

File: packages/stories-block/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# WordPress Stories Dashboard
```
Comment (swissspidy) [inline packages/stories-block/README.md]: Wrong readme content

Comment (spacedmonkey) [inline packages/stories-block/README.md]:
```suggestion
# WordPress Stories Block
```

File: packages/stories-block/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Web Stories dashboard for use in WordPress admin.
```
Comment (spacedmonkey) [inline packages/stories-block/README.md]:
```suggestion
Web Stories Block for use in WordPress editor.
```

**Review by swissspidy — APPROVED** [review]
Neat!

**Pull request merged**